contributors/devel/release/scalability-validation.md

```diff
@@ -0,0 +1,123 @@
+
+- Time-consuming (12-24 hrs)
+- Expensive (tens of thousands of core hours per run)
+- Blocking other large tests (quota limitations + only one large test project available viz. 'kubernetes-scale')
+
+So we don’t want to run them too frequently. On the other hand, running them too infrequently means late identification and piling up of regressions. So we choose the following middleground:  
+(B = release-blocking, NB = not release-blocking)
+
+- Performance tests on 2k-node cluster in GCE/GKE alternatingly each week (NB)
+  - would catch most scale-related regressions
+  - would catch bugs in both GCE and GKE
+- Performance tests on 5k-node cluster in GCE each week (B)
+  - would catch scale-related bugs missed by 2k-node test (should be rare)
+- Correctness tests on 2k-node cluster in GCE/GKE alternatingly each week (NB)
+  - correctness tests failing on just large clusters is rare, so weekly is enough
+  - would catch bugs in both GCE and GKE
+- Correctness tests on 5k-node cluster in GCE/GKE alternatingly each week (B for GCE)
+  - would catch bugs left out by 2k-node (should be rare)
+  - would verify 5k-node clusters on GKE but without blocking the release
+
+Here's the proposed schedule (may be fine-tuned later based on test health / release schedule):
+
+| | Mon | Tue | Wed | Thu | Fri | Sat | Sun
+| ---- | :----: | :----: | :----:| :----: | :----: | :----: | :----:
+| Week 2k | GCE 5k perf (B) | | | GCE 5k corr (B) | | GKE 2k perf+corr (NB) |
+| Week 2k+1 | GCE 5k perf (B) | | | GKE 5k corr (NB) | | GCE 2k perf+corr (NB) |
+
+Why this schedule?
+
+- 5k tests might need special attention in case of failures so they should run on weekdays
+- Running 5k perf test on Mon gives the whole week for debugging in case of failures
+- Running 5k corr test on Thu gives Tue-Wed to debug any setup issues found on Mon
+- Running 5k corr tests alternatingly on GKE and GCE validates both setups
+- 2k perf and corr tests can be run in parallel to save time
+- Running 2k tests on weekend will free up a weekday for the project to run other tests
+
+Why GKE tests?
+
+Google is currently using a single project for scalability testing, on both GCE and GKE. As a result we need to schedule them together. There's a plan for CNCF becoming responsible for funding k8s testing, and GCE/GKE tests would be separated to different projects when that happens, with only GCE being funded by them. This ensures fairness across all cloud providers.
+
+### Concretely define test configuration
+
+This is a relatively minor issue but it is important that we clearly define the test configuration we use for the release. E.g. there was a confusion this time around testing k8s services, machine-type and no. of the nodes we used (we tested 4k instead of 5k due to a CIDR-setup problem). For ref - [#47344] [#47865]. To solve this, we need to document it under the scalability section of the release-notes in features repo, with the following template:
```

Comment:
I agree with documenting the configuration used to validate large cluster performance.  I am, however, against release-note bloat.  In other words, I don't think this belongs in CHANGELOG.md.  

Would prefer to see this as a separately linked document, either within kubernetes/community/sig-scalability, or colocated with other release-related docs (eg: kubernetes/features/release-N/scalability-validation-report.md)